pom.xml

```diff
@@ -276,6 +276,7 @@
             <plugin>
                 <groupId>org.mule.tools.maven</groupId>
                 <artifactId>mule-module-maven-plugin</artifactId>
+                <version>${project.version}</version>
```

Comment:
you should use the verison of the dep in place of the project version

Reply:
Hmm the parent of hte pom is mule-plugin-mgmt-parent-pom. Check if the plugin is defined in the plugin mangement section of the mule-plugin-mgmt-parent-pom. In that case as it is already defined in the parent it shouldn't fail the enforcer.

Reply:
For latest/dry-run the version in `ConfigReleaseVersions` is: `mule_module_maven_plugin="1.9.0-20241126"` and that matches the `pom.xml` version: `<version>1.9.0-20241126</version>`.

I don't want to hardcode the value, is there any scenario when the versions might differ?

Reply:
Yes, a new property like muleModuleMavenPluginVersion and you need to add a new dep entry in the mule-runtime-release config files. 

But as mentioned in htis comment https://github.com/mulesoft/mule-dsl-api/pull/247#discussion_r1862609369 we shouldn;'t use a version and we should try to use the plugin parent

Reply:
That is exactly the property we should use: `muleModuleMavenPluginVersion` I recicled that, it is already declared in our internal config.